topics/single-cell/tutorials/scrna-case-jupyter_basic-pipeline/tutorial.md

```diff
@@ -500,9 +500,9 @@ We still have too many dimensions. Transcript changes are not usually singular -
 Principal components are calculated from highly dimensional data to find the most spread in the dataset. So in our, ```1982``` highly variable gene dimensions, there will be one line (axis) that yields the most spread and variation across the cells. That will be our first principal component. We can calculate the first ```x``` principal components in our data to drastically reduce the number of dimensions.
 
 > <comment-title>1982???</comment-title>
-> Where did the `1982` come from? 
-> 
-> The quickest way to figure out how many highly variable genes you have, in my opinion, is to re-run ```sc.pp.highly_variable_genes``` function with the added parameter ```subset=True```, therefore: ```sc.pp.highly_variable_genes(output_h5ad, subset=True)```. This subsetting removes any nonvariable genes. 
+> Where did the `1982` come from?
```

Comment:
sometimes I wonder the same thing: Eye of the Tiger, Tainted Love, Abracadabra, Chariots of Fire. Truly was a great year.

Reply:
damnit, I shouldn't have changed anything here or anywhere other than the subtopics and one small section of internal linking. Stupid Github